quasar.config.js

```diff
@@ -51,7 +50,7 @@ module.exports = configure(function (/* ctx */) {
         node: 'node20'
       },
 
-      vueRouterMode: 'hash', // available values: 'hash', 'history'
+      vueRouterMode: 'history', // available values: 'hash', 'history'
```

Comment:
Toto ako funguje? resp. co to zmenilo?

Reply:
Tu sa zmenil router mode z hashu co boli v url na normlane slashe, je to lepsie pre SEO a za mna aj prehladnejsie url to tvori